src/navigator/view/ItemView.js

```diff
@@ -1,16 +1,25 @@
 import { isUndefined } from 'underscore';
+import { isTouchDevice } from 'utils/mixins';
 const ComponentView = require('dom_components/view/ComponentView');
 const inputProp = 'contentEditable';
 let ItemsView;
 
 module.exports = require('backbone').View.extend({
-  events: {
-    'mousedown [data-toggle-move]': 'startSort',
-    'click [data-toggle-visible]': 'toggleVisibility',
-    'click [data-toggle-select]': 'handleSelect',
-    'click [data-toggle-open]': 'toggleOpening',
-    'dblclick [data-name]': 'handleEdit',
-    'focusout [data-name]': 'handleEditEnd'
+  events: function() {
+    var eventsList = {
+      'click [data-toggle-visible]': 'toggleVisibility',
+      'click [data-toggle-select]': 'handleSelect',
+      'click [data-toggle-open]': 'toggleOpening',
+      'dblclick [data-name]': 'handleEdit',
+      'focusout [data-name]': 'handleEditEnd'
+    };
+
+    eventsList[
+      (isTouchDevice() ? 'touchstart' : 'mousedown') + ' [data-toggle-move]'
+    ] =
+      'startSort';
+
+    return eventsList;
```

Comment:
I think this will be more easier and readable
```js
'mousedown [data-toggle-move]': 'startSort',
'touchstart [data-toggle-move]': 'startSort',
```

Reply:
I don't remember the exact cause for this . 

We can't set the events together because then the function is called twice with diferent Event objects. This can cause strange effects, also many times these function store data from these events which are used in other listeners ( the data are not consistent if the function can be called from diferent events ) .

I was testing on Chrome, Firefox and IE, usign mouse and touch devices. And listening just for mouse events was not consistent. ( i think the mouse implementation on IE touch devices was the problem )